src/recensio/plone/profiles/default/workflows.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<object name="portal_workflow">
+  <bindings>
+    <type type_id="Discussion Item">
```

Comment:
Do we have discussions on recensio?

Reply:
Yes, on the presentations. (We plan to ask whether we can discard them, but if not then we also need to migrate the comments on them.)